region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -1298,8 +1324,13 @@
       "link": [5, 8],
       "name": "Space Jump Alcatraz Escape",
       "requires": [
-        "canMidAirMorph",
-        "SpaceJump"
+        "canTrivialMidAirMorph",
+        "canCarefulJump",
+        "SpaceJump",
+        {"or": [
+          "HiJump",
+          "canMidAirMorph"
```

Comment:
I think this should remain even with HiJump. Most players at this level won't be able to utilize the extra height that HJB gives without losing the space jump